fu/basic/Fu.py

```diff
@@ -44,8 +44,8 @@ def construct(s, DataType, PredicateType, CtrlType,
     s.recv_const = RecvIfcRTL(DataType)
     s.recv_opt = RecvIfcRTL(CtrlType)
     s.send_out = [SendIfcRTL(DataType) for _ in range(num_outports)]
-    s.send_to_controller = SendIfcRTL(s.CgraPayloadType)
-    s.recv_from_controller = RecvIfcRTL(s.CgraPayloadType)
+    s.send_to_ctrl_mem = SendIfcRTL(s.CgraPayloadType)
```

Comment:
Add comment: Used for command delivery between functional unit and control memory, which is usually sent to (or received from) the controller.